tools/test_env

```diff
@@ -73,6 +73,9 @@ elif [ "CHK$1" == "CHKsriov" ];then
     sudo ip link set eth1 vf 0 vlan 100 mac 36:f7:09:ef:95:f0
     sudo ip link set eth1 vf 1 mac 36:f7:09:ef:95:f1
     sudo ip link set eth1 up
+elif [ "CHK$1" == "CHKtun" ];then
+    sudo ip tuntap add name tun1 mode tap \
```

Comment:
You didn't add `tun1` into `test_env rm` clean up section.

Reply:
Fixed.